apps/adoption/src/app/pages/getting-started.page.html

```diff
@@ -0,0 +1,34 @@
+<h1>Getting Started</h1>
+
+<h3>For who is this guide?</h3>
+<p>
+  This guide is for everyone that want and needs to adopt our new Core components into there existing Clarity Angular
+  application. It's design to help you rework and adopt in the best way and still get the same result as before.
+</p>
+
+<h3>How to use this guide?</h3>
+<p>
+  The guide is structure into sections, every section will provide examples based on our Angular components and there
+  Core counterparts. Into the form of guides, snippets and corner cases that will help you with the adoption of the new
+  components and workflows.
+</p>
+
+<h3>Where to find more information?</h3>
+<p>
+  The guide won't always reflect the latest API changes inside the components so if you want to know more on specific
+  section you could go and check the <a href="https://clarity.design/" target="_blank">official documentation</a>
+</p>
+
+<!--
+<h3>Boilerplates</h3>
```

Comment:
Are we keeping this around or should we remove it?

Reply:
When Stackblitz or Codesandbox start working - here will be the place that you could find "Getting started kit" - A basic working example of Angular App or Core App 

I commented it just because I didn't want to drop the code - I have some deep inside big hopes that this will be a working solution for the moment.